pkg/api/v1beta3/dynakube/extensions.go

```diff
@@ -98,3 +98,15 @@ type ImageRefSpec struct {
 	// Indicates a tag of the image to use
 	Tag string `json:"tag,omitempty"`
 }
+
+func (ref ImageRefSpec) String(defaultRepo, defaultTag string) string {
```

Comment:
do we really need to pass default repo and tag as parameters? Not that it matters a lot but it somehow looks weird because of the word "default"

Reply:
`ImageRefSpec` is not only used by the LogModule, but I agree, there is a better solution

Reply:
done